src/composables/useIsDarkTheme/index.ts

```diff
@@ -3,10 +3,10 @@
  * SPDX-License-Identifier: AGPL-3.0-or-later
  */
 
-import type { DeepReadonly, Ref } from 'vue'
+import type { DeepReadonly, MaybeRefOrGetter, Ref } from 'vue'
 
 import { createSharedComposable, usePreferredDark, useMutationObserver } from '@vueuse/core'
-import { ref, readonly, watch, inject, computed } from 'vue'
+import { computed, inject, ref, readonly, toValue, watch } from 'vue'
```

Comment:
```suggestion
import { createSharedComposable, toValue, usePreferredDark, useMutationObserver } from '@vueuse/core'
import { computed, inject, ref, readonly, watch } from 'vue'
```

Reply:
ah yes `toValue` :facepalm: 
thank you!